packages/client/src/components/Modals/EditAgency.vue

```diff
@@ -44,6 +84,16 @@
             required
           ></b-form-input>
         </b-form-group>
+        <form ref="form" @click="handleDelete">
+          <span id="disabled-wrapper" class="d-inline-block" tabindex="0">
+            <b-button v-bind:disabled="userRole !== 'admin'" style="pointer-events: none;" variant="danger">
+              Admin Delete Agency
+            </b-button>
+          </span>
+          <b-tooltip v-if="userRole !== 'admin'" target="disabled-wrapper" triggers="hover">
+            You cannot delete an agency with children. Reassign child agencies to continue deletion.
```

Comment:
Tooltip msg is kinda confusing. 
Can you not delete an agency b/c you are not an admin? or b/c the agency has children?

Reply:
If the user is not an admin or if the agency has children, the button will be disabled and the tooltip will appear on hover